src/sql/src/plan/statement/show.rs

```diff
@@ -508,7 +510,7 @@ fn show_recorded_views<'a>(
     let mut where_clause = format!("schema_id = {schema_spec}");
 
     if let Some(cluster) = in_cluster {
-        where_clause += &format!(" AND cluster_id = {}", cluster.0);
+        write!(where_clause, " AND cluster_id = {}", cluster.0)?;
```

Comment:
This would be more clear as an `expect`, since `write!` on a `String` can't fail.